functorch/test/test_vmap.py

```diff
@@ -3759,6 +3759,9 @@ def f(e_):
         skip('linalg.multi_dot'),  # accepts list of tensor inputs, has its own special test
         xfail('linalg.vander'),
         xfail('linalg.vecdot'),
+        # Throws in vmap
+        # IndexError: Dimension out of range (expected to be in range of [-1, 0], but got -2)
+        xfail('linalg.matrix_norm'),
```

Comment:
This PR didn't add `linalg.matrix_norm`, right? If so then this is a regression that we should fix before merging

Reply:
This test is xfailed per @samdow's advice https://github.com/pytorch/pytorch/pull/81761#discussion_r948021867

Reply:
Oh, I forgot about the discussion above. This is fine, thanks for checking @ngimel.